docs/ios/geofencing/002.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "id": "https://schemas.datacubed.com/android/geofencing/002.json",
+    "index": "https://schemas.datacubed.com/index.json",
+    "type": "array",
+    "items":    {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "name": {
+            "type": "string"
+          },
+          "event_type": {
+            "type": "string"
+          },
+          "timestamp": {
+              "$ref": "https://schemas.datacubed.com/definitions.json#/timestamp_def"
+          },
+          "last_updated": {
+              "$ref": "https://schemas.datacubed.com/definitions.json#/timestamp_def"
+          },
+          "latitude": {
+              "type": "number",
+              "minimum": -90,
+              "maximum": 90
+          },
+          "longitude": {
+              "type": "number",
+              "minimum": -180,
+              "maximum": 180
+          },
+          "radius": {
+              "type": "number",
+              "minimum": 0
+          },
+          "snoozed": {
```

Comment:
Let's make 'snoozed' optional so our solution is expandable for other geofence use cases as well.

Reply:
check line 42 now